src/models/VisualStyleModel/VisualPropertyName.ts

```diff
@@ -19,6 +19,7 @@ export const NodeVisualPropertyName = {
   NodeMaxLabelWidth: 'nodeMaxLabelWidth',
   NodeZOrder: 'nodeZOrder',
   NodeLabelPosition: 'nodeLabelPosition',
+  PieSize: 'pieSize',
```

Comment:
This should be a more accurate visual property name that corresponds to its CX visual property name